lmcache/utils.py

```diff
@@ -78,3 +80,25 @@ def _lmcache_nvtx_annotate(func, domain="lmcache"):
         color=_get_color_for_nvtx(func.__qualname__),
         domain=domain,
     )(func)
+
+
+def _get_size(
```

Comment:
Can we change the name of this function and add some detailed docstring for it?

Usually, the function starting with `_` is a private function that cannot be seen and called by other modules. However, this function is designed as a utility function that will be called by multiple different modules.